### PR TITLE
Add 2016-05-06 meeting announcement

### PR DESCRIPTION
**Pull request by jflory7**
This commit adds in the latest meeting announcement for our upcoming meeting on May 6th. The meeting topic will be LetsEncrypt and will have a talk, followed by a workshop, followed by an open floor.

@Serubin: Let me know if this looks good to you for this week!

@thenaterhood / @repkam09: If you get a chance to take a look, let me know if this all checks out!

### CODE REVIEW

**Comment by jflory7** [comment]
@jflory7 Does this work?

File: announcements/_posts/2016-05-04-week-14-letsencrypt.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+author: Justin W. Flory
+title: "Spring Week 14: LetsEncrypt"
+layout: post
+---
+
+Welcome to Week 14, RITluggers! Hope everyone's week is going well so far.
+
+In case you missed it at the last meeting, our new Eboard was officially elected at our last meeting. The 2016-2017 President is Justin W. Flory and Vice President is Solomon Rubin. For this week and the next, we'll be getting a headstart into running the club.
```
Comment (thenaterhood) [inline announcements/_posts/2016-05-04-week-14-letsencrypt.md]: The department of redundancy department would like to have a word with you about your first sentence ;)

**Comment by repkam09** [comment]
Could add something about the new Telegram/IRC chats as well to get people involved

**Comment by jflory7** [comment]
@repkam09 Yeah, good point. I'll work on getting that in now.

**Comment by Serubin** [comment]
@jflory7 Looks great, but @thenaterhood's comment is real. =D

**Comment by jflory7** [comment]
Okay, merged in @repkam09's feedback! Everything look good?

**Comment by thenaterhood** [comment]
🏤

**Comment by repkam09** [comment]
Looks good! Do the thing!

**Pull request merged**